packages/reactivity/src/computed.ts

```diff
@@ -15,14 +15,20 @@ export interface WritableComputedRef<T> extends Ref<T> {
   readonly effect: ReactiveEffect<T>
 }
 
-export type ComputedGetter<T> = (...args: any[]) => T
+export type ComputedGetter<T> = () => T
+export type ComputedGetterWithVModel<T> = (...args: any[]) => T
```

Comment:
Let's rename this to `ComputedGetterWithInstance`

Reply:
Done